NuGet.config

```diff
@@ -19,6 +19,7 @@
     <add key="dotnet3.1-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="dotnet5-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json" />
+    <add key="core-setup" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-sym-dotnet-core-setup-a49e99c5/nuget/v3/index.json" />
```

Comment:
This is the symbols feed, not the feed that @MattGal recommended, which is https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-a49e99c5/nuget/v3/index.json. Note the lack of **-sym** in this one.